### DISCUSSION TITLE
Aeotec Ltd. - ZW141 and ZWA019 - how to contribute?

### DISCUSSION THREAD

**Discussion by jprates** [zwave-js/zwave-js#2332] [Q&A]
Hi all,

I had some devices from Aeotec, all reporting back with manufacturer id 0x0086 (AEON Labs) working fine under zwave-js.
Last week I bought more devices (2 new types) from the same manufacturer, and to my surprised they were not recognized by zwave-js.

From the network dump file generated by home assistant I see the new ones use a different manufacturer id 0x0371.
My new devices announced as manufacturer 881 (0x0371) on the z-wave network are:
 Aeotec Nano Shutter  // zw141-c // ("productId": 141 (0x008d), "productType": 3, "firmwareVersion": "3.4")
 Aeotec Water Sensor 7 Pro // zwa019 //  ("productId": 19 (0x0013), "productType": 2, "firmwareVersion": "1.2")

Inside the addon_core_zwave_js container on the 0x0371 folder on /usr/src/node_modules/@zwave-js/config/config/devices these 2 products are missing.
Inside the index.json (serverVersion 1.4.0) there are two  versions of the nano shutter using manufacturerId 0x0086, none of the water sensor 7 pro:
```
        {
                "manufacturerId": "0x0086",
                "manufacturer": "AEON Labs",
                "label": "Nano Shutter Controller",
                "productType": "0x0003",
                "productId": "0x008d",
                "firmwareVersion": {
                        "min": "0.0",
                        "max": "255.255"
                },
                "filename": "0x0086/nano_shutter_controller.json"
        },
        {
                "manufacturerId": "0x0086",
                "manufacturer": "AEON Labs",
                "label": "Nano Shutter Controller",
                "productType": "0x0103",
                "productId": "0x008d",
                "firmwareVersion": {
                        "min": "0.0",
                        "max": "255.255"
                },
                "filename": "0x0086/nano_shutter_controller.json"
        },
```

Final note on this announcement https://github.com/zwave-js/node-zwave-js/issues/1600 to say both devices appear listed:
   Aeotec Ltd. - ZW141 - Nano Shutter
   Aeotec Ltd. - ZWA019 - Water Sensor 7 Pro

I've been using other devices whose json files were already available thanks to someone elses work, so I'm willing to give my two cents to the project and add these new two.
Anyone care to show me the right direction, and hopefully I will be able to create some nicely formatted and technically correct json files?

Cheers all,
-jprates

**Comment by AlCalzone** [comment]
@blhoward2
